client/src/components/LevelLayout.tsx

```diff
@@ -50,6 +56,19 @@ const LevelLayout: React.FC<Props> = ({
         {showInput && <StepValidation />}
         {children}
       </Container>
+      <Flex>
+        <Spacer />
+        <Box
+          display="flex"
+          alignItems="center"
+          bg="gray.200"
+          p={4}
+          borderWidth="1px"
```

Comment:
You might be missing `borderStyle` and `borderColor`. `borderWidth` alone has no effect.

Reply:
Resolved on last push